content/blog/resource-coverage-in-the-browser/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Resource coverage in the browser"
+date: "2020-06-17"
+description: "Get details of unused CSS/JS in the devtools."
+tags:
+  - browser
+  - javascript
+  - performance
+show: true
+author: trishul
+---
+
+One of the performance wins for the websites is the deferred loading of JS or CSS which are not required at the first load.
+While starting a project it's easy to maintain this, over the period of time when there are consistent developments and multiple devs contributing to the project, it may not be that smooth.  
```

Comment:
```suggestion
it's easy to maintain this, over the period when consistent developments and multiple devs are contributing to the project, it may not be that smooth.
```